Cargo.toml

```diff
@@ -41,7 +41,7 @@ tokio = { version = "1.35.1", default-features = false }
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
 up-rust = "0.3.0"
-zenoh = { version = "1.0.0" }
+zenoh = { version = "1.2.1" }
```

Comment:
I assume that a Zenoh 1.0.0 client can still connect to a 1.2.1 server?

Reply:
I believe it should be.